### ISSUE TITLE
sdwan_sla_class_policy_object, fallback_best_tunnel_* items can be 0

### ISSUE THREAD

**Issue by cmohorea** [CiscoDevNet/terraform-provider-sdwan#315]
It could be more of vManage problem but still:
"fallback_best_tunnel_jitter", "fallback_best_tunnel_latency", "fallback_best_tunnel_loss" parameters may be skipped during config.

![image](https://github.com/user-attachments/assets/d9216c03-0d3e-41a0-94da-02dd8d8ebe48)

When they are unused, in terraform they show up as 0 (unlike main parameters (loss, jitter, latency) which show up as null) which probably make sense in this case, since variance of 0 actually means that no variance in play.
However, 0 is not accepted in the terraform configuration (allowed range starts at 1), so they cannot be configured as "0"
They can be set as "null" with terraform (but I am not sure what the actual vManage config would be, it could actually require 0 there) but after editing in GUI, they will be set to 0 and terraform would try to fix it on the next run:
```
Terraform will perform the following actions:

  # sdwan_sla_class_policy_object.test_CN_SLA_DEFAULT will be updated in-place
  ~ resource "sdwan_sla_class_policy_object" "test_CN_SLA_DEFAULT" {
      - fallback_best_tunnel_jitter   = 0 -> null
      - fallback_best_tunnel_loss     = 0 -> null
```

I believe that it should be allowed to be 0, and documented that 0 means it's not in use.

**Comment by cmohorea** [comment]
```
resource "sdwan_sla_class_policy_object" "test_CN_SLA_DEFAULT" {
  name = "test_CN_SLA_DEFAULT"
  fallback_best_tunnel_criteria = "loss-latency"
  fallback_best_tunnel_latency = 10
#  fallback_best_tunnel_jitter = null
#  fallback_best_tunnel_loss = null
  latency = 200
  loss = 10
}
```
config for the reference

**Comment by cmohorea** [comment]
it is fine to not allow 0 - that was extra troubleshooting info. null is accepted in YAML and seems to have the same effect as omitting the setting (expected, I guess)

I see that when template is created by TF, those 3 fields are missing, so I guess they are assumed to be null by vManage.... not sure how they become "0" when doing "terraform plan". I guess they should be always included in the TF's API payload and/or assumed Null vs 0 when missing... you know this better!
